index.html

```diff
@@ -0,0 +1,176 @@
+        </a>
+        <ul class="menu desktop-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">TS Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section class="hero">
+      <q>Hello! Tech World!</q>
+      <h1>TECHSPLENDOR SYMPOSIUM 2023</h1>
+      <p class="intro">
+        TechSplendor, the pinnacle of technological excellence! Prepare to immerse yourself in an extraordinary tech
+        journey like never before.
+      </p>
+      <p class="schedule">2023.10.16(MON) ~ 23rd(FRI)</p>
+      <p class="venue">
+        @ Mayor's Garden of Lira, Coronation Park and more
+      </p>
+    </section>
+
+    <section class="main-program">
+      <h2>Main Program</h2>
+      <hr>
+      <div class="programs">
+        <article>
+          <img src="./images/main/program-1.png" alt="Picture of first program">
+          <h3>Expert Talks</h3>
+          <p>Listen to experts, be inspired by their journeys and learn from their invaluable experiences.</p>
+        </article>
+        <article>
+          <img src="./images/main/program-2.png" alt="Picture of second program">
+          <h3>Interactive Workshops</h3>
+          <p>Dive into hands-on workshops that cater to all skill levels, from coding novices to seasoned developers.
+          </p>
+        </article>
+        <article>
+          <img src="./images/main/program-3.png" alt="Picture of third program">
+          <h3>Challenges & Competitions</h3>
+          <p>Put your skills to the test in exhilarating tech challenges and competitions</p>
+        </article>
+        <article>
+          <img src="./images/main/program-4.png" alt="Picture of fourth program">
+          <h3>Innovation Showcases</h3>
+          <p>Witness the future come to life in our innovation showcases by cutting-edge startups and tech companies</p>
+        </article>
+        <article>
+          <img src="./images/main/program-5.png" alt="Picture of fifth program">
+          <h3>Networking - Collaboration</h3>
+          <p>Connect with fellow tech enthusiasts, entrepreneurs, and industry experts.</p>
+        </article>
+      </div>
+      <a href="#" class="join">Join TechSplendor 2023</a>
+      <a href="#" class="see-all">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section class="experts">
+      <h2>Meet Our Experts</h2>
+      <hr>
+      <ul class="expert-list">
+      </ul>
+      <button class="show-more">
+        <span>MORE <img src="./images/experts/down-arrow.svg" alt="Show more icon"></span>
+      </button>
+      <button class="show-less">
+        <span>LESS <img src="./images/experts/down-arrow.svg" alt="Show less icon" class="up-arrow"></span>
+      </button>
```

Comment:
Original design
![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/78251927/b237ae74-9fcd-4686-81b4-93460f02fa06)

### Mobile
![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/78251927/44150af0-f8d9-420b-a7e1-098a163d5a80)

Your design
![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/78251927/257a977a-5108-4697-b8b9-e694f711253a)

[OPTIONAL] - Please, let's remove the button `MORE` and `LESS` on the desktop version of your website as it is only required on the mobile design, according to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 🤗